From c8f986db4de0a1b90f2bef3821d7dc601c0b6601 Mon Sep 17 00:00:00 2001
From: Eidolon <furyhunter600@gmail.com>
Date: Tue, 30 Sep 2025 20:18:47 -0500
Subject: [PATCH] Remove unintended g++ warnings

---
 src/CMakeLists.txt | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 35e8cb1ca..a5462ee1b 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -622,11 +622,10 @@ target_compile_options(SRB2SDL2 PRIVATE
 	# C++, GNU
 	$<$<AND:$<COMPILE_LANGUAGE:CXX>,$<CXX_COMPILER_ID:GNU>>:
 		-Wall
-		-Wno-c++98-compat
 		-Wno-c++11-compat
 		-Wno-c++14-compat # No C++14 compat needed
-		-Werror=c++20-extensions
-		-Werror=c++23-extensions # Disallow newer language features entirely
+		-Wc++20-extensions
+		-Wc++23-extensions # Disallow newer language features entirely
 	>
 
 	# C++, Clang and Apple Clang
-- 
2.45.2.windows.1

